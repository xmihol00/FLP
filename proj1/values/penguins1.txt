39.5,17.4,186.0,3800.0,0.0
40.3,18.0,195.0,3250.0,0.0
39.3,20.6,190.0,3650.0,1.0
39.2,19.6,195.0,4675.0,1.0
41.1,17.6,182.0,3200.0,0.0
42.5,20.7,197.0,4500.0,1.0
46.0,21.5,194.0,4200.0,1.0
37.8,18.3,174.0,3400.0,0.0
37.7,18.7,180.0,3600.0,1.0
35.9,19.2,189.0,3800.0,0.0
38.2,18.1,185.0,3950.0,1.0
38.8,17.2,180.0,3800.0,1.0
35.3,18.9,187.0,3800.0,0.0
37.9,18.6,172.0,3150.0,0.0
39.5,16.7,178.0,3250.0,0.0
37.2,18.1,178.0,3900.0,1.0
39.5,17.8,188.0,3300.0,0.0
40.9,18.9,184.0,3900.0,1.0
36.4,17.0,195.0,3325.0,0.0
39.2,21.1,196.0,4150.0,1.0
38.8,20.0,190.0,3950.0,1.0
42.2,18.5,180.0,3550.0,0.0
37.6,19.3,181.0,3300.0,0.0
39.8,19.1,184.0,4650.0,1.0
36.5,18.0,182.0,3150.0,0.0
40.8,18.4,195.0,3900.0,1.0
36.0,18.5,186.0,3100.0,0.0
44.1,19.7,196.0,4400.0,1.0
37.0,16.9,185.0,3000.0,0.0
39.6,18.8,190.0,4600.0,1.0
41.1,19.0,182.0,3425.0,1.0
36.0,17.9,190.0,3450.0,0.0
42.3,21.2,191.0,4150.0,1.0
