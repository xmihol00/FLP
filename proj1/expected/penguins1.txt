Torgersen
Torgersen
Torgersen
Torgersen
Torgersen
Torgersen
Torgersen
Torgersen
Biscoe
Biscoe
Biscoe
Biscoe
Biscoe
Biscoe
Biscoe
Dream
Dream
Dream
Dream
Dream
Dream
Dream
Dream
Dream
Dream
Dream
Dream
Dream
Dream
Dream
Dream
Dream
Dream
Dream
